desktop-app/src/main/main.ts

```diff
@@ -177,6 +177,30 @@ const createWindow = async () => {
     )}`
   );
 
+  const isWindows = process.platform === 'win32';
```

Comment:
LGTM but @manojVivek can you take a look too? I am not able to test it on windows, I tried it on mac and it doesn't break anything.